Shared/sdk/version.h

```diff
@@ -1,10 +1,13 @@
 //
-// MTA10_Server/version.h
-//
-// Server version file
+// Shared/version.h
 //
 #pragma once
 
+//
+// To compile a client:
+//      1. set MTASA_VERSION_TYPE to VERSION_TYPE_CUSTOM
+//      2. Use netc.dll from the latest unstable build (nightly.mtasa.com)
```

Comment:
This comment looks wrong.. the netc.dll made for custom builds (from install-data.bat), which is located at [https://mirror.mtasa.com/bdata/netc.dll](https://mirror.mtasa.com/bdata/netc.dll), should be used.

Reply:
That's the current comment on master (https://github.com/multitheftauto/mtasa-blue/blob/master/Client/version.h#L10)

What would be an accurate fix?

```suggestion
//      2. Run install-data for netc.dll
```

Reply:
@Dutchman101 Any comment on this?

Reply:
@Dutchman101 Any comment on this?

Reply:
To correct the existing comment(s) that are incorrect.. either in this PR or a new one. Like just refer to the existing build instructions or install-data without specific mention of netc. I would suggest to just refer to the build instructions readme.